IFC4x3/Properties/r/RatedFrequency_0xk7ZvULjDVACUV9Qs5Q4T/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RatedFrequency_0xk7ZvULjDVACUV9Qs5Q4T" Name="RatedFrequency" UniqueId="3bb878f9-795b-4d7c-a31e-7c96b615a11d" PrimaryDataType="IfcFrequencyMeasure" />
```

Comment:
Possible reuse of `NominalFrequency`?

Reply:
Agree. Commit : b5e658cad21e4944423ed95d945268c722299c68